sdk/python/feast/infra/online_stores/sqlite.py

```diff
@@ -432,6 +434,7 @@ def retrieve_online_documents(
                 _build_retrieve_online_document_record(
                     entity_key,
                     string_value if string_value else b"",
+                    # This may be a bug
```

Comment:
is the intention to leave this comment?

Reply:
Actually yes, I need to resolve it in a follow up PR